test/scenarios/Form Test - Smooth Operators.crank.yml

```diff
@@ -0,0 +1,30 @@
+scenario: Smooth Operators Test in Marketo
+description: |
+  This scenario proves that the smooth operators when validating fields are working as
+  expected.
+
+tokens:
+  test:
+    email: cjdeclaro@thisisjust.atomatest.com
+    lastname: De Claro
+    firstname: CJ
+    company: Yeah corp
+    title: Executive
+    country: United States
+    submit: button[type=submit]
+  
+steps:
+- step: Given I navigate to http://go.automatoninc.com/dev-qa-basic-full-form.html
+- step: When I fill out input[id=FirstName] with {{test.firstname}}
+- step: And I fill out input[id=LastName] with {{test.lastname}}
+- step: And I fill out input[id=Email] with {{test.email}}
+- step: And I fill out input[id=Title] with {{test.title}}
+- step: And I fill out input[id=Company] with {{test.company}}
+- step: And I fill out select[id=Country] with {{test.country}}
+- step: And I submit the form by clicking {{test.submit}}
+  failAfter: 60
+- step: Then the email field on marketo lead {{test.email}} should not be cjdeclaro@atomatest.com
+- step: And the lastName field on marketo lead {{test.email}} should contain de
+- step: And the company field on marketo lead {{test.email}} should not contain Mkto
+- step: And the title field on marketo lead {{test.email}} should be {{test.title}}
```

Comment:
Thanks @roanlibelo!  Exercising all of the operators is great.

- [ ] I'd recommend also exercising the greater than / less than operators using the `createdAt` lead field and dates that are really far in the past / future (e.g. `1900-01-01T00:00:00` and `2100-01-01T00:00:00`